tests/general-api-tests/layers.spec.ts

```
@@ -39,19 +39,25 @@ languagesToTestLayers.forEach((language) => {
+      ).toBeGreaterThan(0);
       const linesUrl = firstObject.source.urls[0];
       expect(linesUrl, "Lines url should be defined").toBeDefined();
 
       // Check that atlasLayersUrl gave correct translation
-      expect(linesUrl).toContain(`style_ninja_${language}.json`);
+      expect(
+        linesUrl,
+        `Layers url should contain style_ninja_${language}.json`
+      ).toContain(`style_ninja_${language}.json`);
 
       const responseLines = await request.get(linesUrl!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)